docs/migration-guide/to-15.md

```diff
@@ -124,10 +124,7 @@ We've removed the deprecated rules from the latest version of the [standard conf
 
 There are lots of other rules we don't turn on in the [standard config](https://www.npmjs.com/package/stylelint-config-standard) and you can learn more about using them to customize Stylelint to your exact needs in our [new guide](../user-guide/customize.md).
 
-Alternatively, you can continue to enforce stylistic consistency with Stylelint by using one of the community plugins that have migrated the deprecated rules:
-
-- [stylelint-stylistic](https://www.npmjs.com/package/stylelint-stylistic)
-- [stylelint-codeguide](https://www.npmjs.com/package/stylelint-codeguide)
+Alternatively, you can continue to enforce stylistic consistency with Stylelint by using the community plugins [stylelint-stylistic](https://www.npmjs.com/package/@stylistic/stylelint-plugin) that have migrated the deprecated rules.
```

Comment:
```suggestion
Alternatively, you can continue to enforce stylistic consistency with Stylelint by using the community plugin [`@stylistic/stylelint-plugin`](https://www.npmjs.com/package/@stylistic/stylelint-plugin) that has migrated the deprecated rules.
```

Reply:
My comment about the package name: https://github.com/stylelint/awesome-stylelint/pull/62#pullrequestreview-1791073624